authors/apps/articles/models.py

```diff
@@ -1,7 +1,14 @@
+import readtime
```

Comment:
@cob04, why are you importing `readtime` here?

Reply:
let me clean it up and run tests.

Reply:
actually readtime is necessary here,  tests are failing.

Reply:
@cob04, do you mean that readtime as a dependency was not added in the test file?

Reply:
I did not add it, it was there already(at `models.py`) but it got displaced when added my dependency imports.